config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -653,3 +653,38 @@ presubmits:
           limits:
             cpu: "4"
             memory: "4Gi"
+
+  - name: pull-etcd-fuzzing-v3rpc
+    optional: true # remove once the job is stable
+    cluster: eks-prow-build-cluster
+    always_run: false # remove once the job works
+    branches:
+      - main
+    decorate: true
+    decoration_config:
+      timeout: 60m
+    annotations:
+      testgrid-dashboards: sig-etcd-presubmits
+      testgrid-tab-name: pull-etcd-fuzzing-v3rpc
+    spec:
+      containers:
+      - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250205-f1f3519e6b-master
+        command:
+          - runner.sh
+        args:
+          - bash
+          - -c
+          - |
+            return_code=0
+            GOARCH=amd64 CPU=4 make fuzz || return_code=$?
```

Comment:
amd64? But I do not see this test is assigned an amd64 environment. It should be arch agnostic?

Reply:
If we don't explicitly add a `nodeSelector`, it defaults to AMD64. In other words, it only schedules on ARM64 if we add the ARM64 `nodeSelector`.

For example, this is the definition of the AMD64 unit tests job:
https://github.com/kubernetes/test-infra/blob/f414bfaa8700111b528079226ac3e26fbbe61011/config/jobs/etcd/etcd-presubmits.yaml#L29-L55

And this is the same for ARM64, with emphasis on `nodeSelector` at the bottom:
https://github.com/kubernetes/test-infra/blob/f414bfaa8700111b528079226ac3e26fbbe61011/config/jobs/etcd/etcd-presubmits.yaml#L57-L85

Reply:
Just a minor comment: we can just remove `GOARCH=amd64`.

Reply:
Done